### DISCUSSION TITLE
how to get  value from attribute that not exist in config but available in the device, ex: VRC43 and HC2HeatPump?

### DISCUSSION THREAD

**Discussion by pdigerp** [john30/ebusd#1157] [Q&A]
Hello,

I have setup an ebusd container and configured MQTT  on Raspberry 4 to read data from my Vaillant Gas system and it works well.

I have a question :
My VRC430 can provide me very interesting information, example about HC1. But, how can I configure ebusd to provide information about HC2, example HC2HeatPump ?

Thanks,
Pino

**Comment by Koky05** [comment]
VRC430 is mostly one zone controller so if you have more zones (HC2 etc.) you probably have some module like VR61.
For example I have VRC470F and I read all data about Hc1 from 15.f47.csv but for second circuit I use V61 and all data for HC2 I read from 50.v61.mc.csv

**Reply by pdigerp** [reply]
I confirm your answer,

Thanks Koky05
